### ISSUE TITLE
Coffescript or Js

### ISSUE THREAD

**Issue by Willardgmoore** [Travelbuddyapp/travel_buddy_website#13]
We need to decide first. Then we need to convert one to the other.

**Comment by bondparkerbond** [comment]
I vote Javascript and Haml.  or erb, just no Coffee, coffee is dead to me.
ES2015 makes coffee so obsolete it isn't even funny.

**Comment by dapperdandev** [comment]
I agree that coffee is dead. Typescript may be something to look into. I haven't used it, personally, but all of the feedback is positive.
